docs/howto/retry_stalled_jobs.rst

```diff
@@ -0,0 +1,30 @@
+Retry stalled jobs
+==================
+
+When a worker gets a ``SIGINT`` or ``SIGTERM`` signal requesting it to terminate it
+waits for running jobs to finish before actually exiting. But, if the worker gets
+a second ``SIGINT`` or ``SIGTERM`` signal, or if it's killed with ``SIGKILL``, it will
+terminate immediately, possibly leaving jobs with the ``doing`` status in the queue.
+And, if no specific action is taken, these *stalled* jobs will remain in the queue
+forever, and their execution will never resume.
+
+To address that problem, Procrastinate offers functions that can be used in a periodic
+task for retrying stalled jobs. Here's an example:
+
+.. code-block:: python
+
+    @app.periodic(cron="*/10 * * * *")
+    @app.task(queueing_lock="retry_stalled_jobs", pass_context=True)
+    async def retry_stalled_jobs(context, timestamp):
+        stalled_jobs = await app.job_manager.get_stalled_jobs(nb_seconds=30)
```

Comment:
can `30` be a constant with a clear name ?

Reply:
Done, but feel free to suggest a better variable name.